renderer/index.js

```diff
@@ -533,11 +534,25 @@ function closePlayer () {
   if (state.window.isFullScreen) {
     dispatch('toggleFullScreen', false)
   }
+
+  unblockPowerSave()
```

Comment:
I'd make this a comment in the code.

Also, I understand that we have to restart power saving mode when the player window exits, not just when the video is paused, but could we just say `ipcRenderer.send('closing-player')` here or smth similar? (Or just catch an event in the main process that fires when the app is about to close?) Either way, I think OS integrations like this are best handled from the main process.

This is a nitpick tho and not a big deal.